CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 - Updates the `Checkbox` component with an "indeterminate" state through the `isIndeterminate` prop.
 - Updates the `CheckboxGroup` component story with an "indeterminate" state example.
 - Updates the `CheckboxGroup` and `RadioGroup` components to use the `Fieldset` component.
+- Updates the `CheckboxGroup` and `RadioGroup` components to use "fit-content" for their form input elements' width value.
```

Comment:
Not sure this is necessary.  This change is just making the elements behave better.